google-cloud-tools-plugin/src/com/google/cloud/tools/intellij/appengine/facet/AppEngineFacetEditor.java

```diff
@@ -172,6 +174,10 @@ public String getHelpTopic() {
   @Override
   public void onFacetInitialized(@NotNull Facet facet) {
     AppEngineWebIntegration.getInstance().setupDevServer();
+
+    UsageTrackerProvider.getInstance()
+        .trackEvent(GctTracking.APP_ENGINE_ADD_FACET)
```

Comment:
An example I just found is when you start IJ and AppEngineFacet() is called, even though you aren't adding any facet (the IDE is constructing the project and constructing an AppEngineFacet). You'd be counting each time the browser is started as a new facet added.

I'm afraid there are other cases where AppEngineFacet() is called but a facet isn't being added by a user that I'm not seeing.